extension/js/background.js

```diff
@@ -1,20 +1,24 @@
 var MILLIS_BEFORE_CLEAR = 1000 * 60; // 60 seconds
 var CLEAR_DELAY = 20000;
 var MAX_URL_LEN_SHOWN = 50;
-var LT = function(a,b) {return a < b};
-var GT = function(a,b) {return a > b};
-var LT_OBJ = function(a,b) {
+var LT = function (a, b) {
+    return a < b
+};
+var GT = function (a, b) {
+    return a > b
+};
+var LT_OBJ = function (a, b) {
     return a.time < b.time;
 }
 
 var db
 
-var GT_OBJ = function(a,b) {
+var GT_OBJ = function (a, b) {
     return a.time > b.time;
 }
 
-Array.max = function( array ){
-    return Math.max.apply(Math,array);
+Array.max = function (array) {
```

Comment:
augmenting objects is usually not a good idea. I also think this is not used anywhere so it can probably be removed